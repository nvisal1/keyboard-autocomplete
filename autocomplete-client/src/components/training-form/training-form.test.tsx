import React from 'react';
import renderer from 'react-test-renderer';

import TrainingForm, { TrainingFormProps } from './training-form';

describe('When the TrainingForm component is created', () => {
    it('should render correctly', () => {
        const requiredProps = getRequiredProps();
        const currentSnapshot = renderer.create(<TrainingForm { ...requiredProps }></TrainingForm>).toJSON();

        expect(currentSnapshot).toMatchSnapshot();
    });
});

function getRequiredProps(): TrainingFormProps {
   const handleSubmit = (_: string) => {}; 

   return { handleSubmit };
}