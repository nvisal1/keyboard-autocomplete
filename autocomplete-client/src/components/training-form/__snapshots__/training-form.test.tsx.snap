// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When the TrainingForm component is created should render correctly 1`] = `
<form
  className="training-form"
  onSubmit={[Function]}
>
  <textarea
    className="training-form__textarea"
    cols={30}
    name="training-passage"
    onChange={[Function]}
    rows={10}
  />
  <button
    className="training-form__submit-button"
  >
    Submit
  </button>
</form>
`;
